man/mono.1

```diff
@@ -1793,6 +1793,9 @@ passed to unmanaged code, and no references kept in managed code,
 which will garbage collect the code.  With this option it is possible
 to track down the source of the problems. 
 .TP
+\fBllvm-disable-self-init\fR
+Disable self-initing LLVM code, rely on PLT machinery (like fullAOT without LLVM).
```

Comment:
(bikeshed) Maybe something like:

> With LLVM codegen, this option will cause methods to be called indirectly through the PLT (As they are in other FullAOT modes, without LLVM).  The indirection allows class constructors to be executed before the first time that any method of a given class is called.   Without this option, LLVM codegen will insert code at the beginning of every method to check the initialization status and call the class constructors, if necessary.  With this option, LLVM will do a slightly worse job of code generation, but the result will be slightly smaller.  Without this option, LLVM may be able to optimize more aggressively, but the code may be bigger.

Reply:
This should just be a debugging option, not something we support for users.

Reply:
@vargaz I agree. I was thinking of taking @lambdageek 's comment in the source and removing it from the man page.